fio.1

```diff
@@ -781,6 +781,12 @@ zone have been read (read workloads). This parameter is valid only for
 sequential workloads and ignored for random workloads. For read workloads,
 see also \fBread_beyond_wp\fR.
 
+.TP
+.BI zone_append
+For \fBrw\fR=write and \fBrw\fR=randwrite requests if zone_append is enabled,
+the request points at the start LBA of the zone. On successful completion
+the device will return the offset where the data has been placed.
```

Comment:
See previous comment.

Reply:
Sure, will be modified as per the comment.